SingularityUI/app/components/common/Navigation.jsx

```diff
@@ -84,9 +84,6 @@ const Navigation = (props) => {
         </div>
         <div className="collapse navbar-collapse" id="navbar-collapse">
           <ul className="nav navbar-nav">
-            <li className={classnames({active: isActive('/', fragment)})}>
```

Comment:
If `Requests` is the default tab now, we should reorder the nav items so that `Status` isn't first.

Reply:
Yeah we can do that as well